content-templates/default-introduction-unit.md

```diff
@@ -57,6 +57,6 @@ TODO: add your activities
 
  -->
 ## What is the main goal?
-By the end of this session, you'll be able to <TODO>
+By the end of this session, you'll be able to {TODO}
```

Comment:
Here and elsewhere, replaced `<>` with `{}`, because the former gets rendered as invalid HTML.